### PR TITLE
fix:delete event twice

### PR DESCRIPTION
**Pull request by chenjinxuan**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by chenjinxuan** [comment]
fix send delete event twice

File: internal/ingress/controller/event_handler.go [hunk 1]
```diff
@@ -61,10 +61,6 @@ func (reh ResourceEventHandler) OnDelete(obj interface{}) {
 	if !reh.IsValidIngresClass(object) {
 		return
 	}
-	reh.UpdateCh.In() <- Event{
```
Comment (chenjinxuan) [inline internal/ingress/controller/event_handler.go]: There is no need to send two delete events here

Comment (chenjinxuan) [inline internal/ingress/controller/event_handler.go]: There is no need to send two delete events here

**Comment by hbagdi** [comment]
Please fill in the contributor form to claim your Kong swag: https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt

**Pull request merged**